integration/nemo/examples/supervised_fine_tuning/jobs/templates/server/config/megatron_gpt_sft.yaml

```diff
@@ -0,0 +1,126 @@
+name: megatron_gpt_sft
+
+trainer:
+  devices: 1
+  accelerator: gpu
+  num_nodes: 1
+  precision: 16
+  logger: False # logger provided by exp_manager
+  enable_checkpointing: False
+  replace_sampler_ddp: False
+  max_epochs: 1
+  max_steps: 200 # consumed_samples = global_step * micro_batch_size * data_parallel_size * accumulate_grad_batches
+  log_every_n_steps: 10 # frequency with which training steps are logged
+  val_check_interval: 50 # If is an int n > 1, will run val every n training steps, if a float 0.0 - 1.0 will run val every epoch fraction, e.g. 0.25 will run val every quarter epoch
+  gradient_clip_val: 1.0
+
+exp_manager:
+  explicit_log_dir: null
+  exp_dir: null
+  name: ${name}
+  create_wandb_logger: False
+  create_tensorboard_logger: True
+  resume_if_exists: False
+  resume_ignore_no_checkpoint: True
```

Comment:
`create_checkpoint_call: False` should be used to not save models on the clients by default.

Reply:
added

Reply:
updated